misc/scripts/install-local.sh

```diff
@@ -251,6 +258,12 @@ fi' | sudo tee "$SRCDIR/$name-pacstall/DEBIAN/postrm" >"/dev/null"
 	fi
 	sudo --preserve-env=PACSTALL_INSTALL dpkg -i "$SRCDIR/$name-pacstall.deb" > "/dev/null"
 	sudo rm "$SRCDIR/$name-pacstall.deb"
+	if ! [[ -d /etc/apt/preferences.d/ ]]; then
```

Comment:
I still think this is only needed once, but I wasn't able to test it

Reply:
Yeah, this bit looks duplicated unnecessarily. @Henryws's comment on this?